packages/@ember/-internals/glimmer/tests/integration/application/engine-test.js

```diff
@@ -928,7 +930,7 @@ moduleFor(
     ) {
       assert.expect(2);
       let tmpl =
-        '{{#link-to "author" 1337 class="author-1337"}}Author 1337{{/link-to}}{{#link-to "author" 1 class="author-1"}}Author 1{{/link-to}}';
+        '<LinkTo @route="author" @model=1337 class="author-1337">Author 1337</LinkTo><LinkTo @route="author" @model=1 class="author-1">Author 1</LinkTo>';
```

Comment:
Not that it super matters but this probably should've been translated as ={{1337}}

Reply:
Same for a couple other similar occurrences